src/v2/models/AppState.js

```diff
@@ -77,6 +78,19 @@ export default Model.extend({
     return this.get('idx').neededToProceed.find((remediation) => remediation.name === formName);
   },
 
+  getRemediationAuthenticationOptions(formName) {
+    const form = this.hasRemediationObject(formName);
+    if (!form) {
+      return false;
```

Comment:
Now that this isn't returning a `boolean`, let's return an empty array here to ensure a uniform type.